Formula/gdal.rb

```diff
@@ -145,6 +146,10 @@ def install
       ENV.delete "HOMEBREW_SDKROOT"
     end
 
+    # Use macOS SDK for headers for 10.14 and above (SDK version must match OS version)
+    ENV["HOMEBREW_SDKROOT"] = MacOS::CLT.sdk_path(MacOS.version) if MacOS.version >= :mojave && MacOS::CLT.installed?
+    ENV["SDKROOT"] = MacOS::CLT.sdk_path(MacOS.version) if MacOS.version >= :mojave && MacOS::CLT.installed?
+
```

Comment:
These are removed right above, I think removing line 142 to 147 fixes it as well.

Reply:
I tried to remove those lines which delete those environment variables, but the problem is still there.